foundation/foundation-mda/src/main/resources/templates/data-delivery-spark/pipeline.pom.xml.vm

```diff
@@ -118,6 +118,7 @@
         <dependency>
             <groupId>org.technologybrewery.krausening</groupId>
             <artifactId>krausening</artifactId>
+            <version>${version.krausening}</version>
```

Comment:
Q: Aren't these versions already set and inherited from the aiSSEMBLE root pom?

Reply:
Good catch. I thought I'd only added them where I had errors, but must have gotten overzealous here.